Makefile

```diff
@@ -35,3 +35,16 @@ checkstyle: black lint
 
 .PHONY: test
 test: only-test checkstyle
+
+VENV = qem-bot
+.PHONY environment:
+	if [ ! -d "$(VENV)" ]; then virtualenv -p python3 $(venv); fi
+	. $(VENV)/bin/activate && pip install -r requirements-dev.txt -r requirements.txt
+
+# devel: environment
+#   maybe use Makefile.VENV instead to get a shell with virtualenv
+# 	# we need to detect what shell we are using
+# 	shell=$$(basename $$SHELL); \
+# 	echo "Activating virtualenv for $$shell"; \
+# 	. $(VENV) && \
+# 	exec $($(SHELL))
```

Comment:
Not really, updated the comment though; Not adding another TODO, to avoid causing a heart attack to @okurz

Reply:
@okurz thank you for the recommendation, however same comment applies:

https://github.com/openSUSE/qem-bot/pull/154#discussion_r1473869498

Reply:
It would make sense to at least state why this code has been disabled; e.g. why it is not good/useful enough for general use and in what situations it would make sense to use it nevertheless. The comments

```
# Developers have bad memory, so we need to remind them to activate the virtualenv
# maybe use Makefile.VENV instead to get a shell with virtualenv
```

don't make that clear to me at all.

Additionally, also if that code was not commented-out I'd frankly struggle to make sense of its intended use and purpose. So that should probably be clarified anyway.

Reply:
> Additionally, also if that code was not commented-out I'd frankly struggle to make sense of its intended use and purpose. So that should probably be clarified anyway.

Good point, updated the comment